draft-cms-masque-ip-proxy-reqs.md

```diff
@@ -133,30 +143,30 @@ packets, in their unmodified entirety. The protocol will support both IPv6
 
 ## Maximum Transmission Unit
 
-The protocol will allow endpoints to negotiate the Maximum Transmission Unit
-(MTU) in use over a given Data Transport. This will allow avoiding IP
-fragmentation, especially as IPv6 does not allow IP fragmentation by nodes
+The protocol will allow endpoints to inform each other of the Maximum
+Transmission Unit (MTU) they are willing to forward. This will allow avoiding
+IP fragmentation, especially as IPv6 does not allow IP fragmentation by nodes
 along the path.
 
 ## IP Assignment
 
-Both the client or server may request to be assigned an IP address range. In
-response to the request, the peer will respond with an IP address range of its
-choosing.
+The server will assign an IP address of its choosing to the client. The client
```

Comment:
This unfortunately is incompatible with the network-to-network case, where the server does not a priori know that the IPs on the remote side are, and needs to accept advertisements from the client. In fact, the client may not have an IP to assign, as it is acting as a "router" for the remote network (and have its own IP assigned already)

Reply:
OK Rephrased to "will be able to"